echopype/tests/consolidate/test_consolidate_integration.py

```diff
@@ -340,7 +340,7 @@ def _tests(ds_test, location_type, nmea_sentence=None):
                 'splitbeam/DY1801_EK60-D20180211-T164025_angles_T4.mat',
                 'splitbeam/DY1801_EK60-D20180211-T164025_angles_T5.mat'
             ],
-            "CW", "power", False, True
+            "CW", "power", False, False
```

Comment:
So this has to be `True` because here we need to test whether both `to_disk=True` and `to_disk=False` are working. This is the only case where `to_disk=True` is tested.

The reason why this test fails right now is because you are doing `source_Sv_type = get_file_format(source_Sv)` after `source_Sv = open_source(source_Sv, "dataset", storage_options)`, so `source_Sv` is no longer a path/str when you do `get_file_formate`.

Good to be careful about changing existing testing parameter combination, since they are usually there for a reason.

Also suggest changing the `write_Sv_to_file` parameter to just `to_disk` to make the code easy to read.

Reply:
@leewujung Ah, sorry, misunderstood the code point there. Updated now!